toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1042,6 +1049,16 @@ void CellArea::drawLevelCell(QPainter &p, int row, int col, bool isReference) {
   TXshCell cell = xsh->getCell(row, col);
   TXshCell prevCell;
 
+  // If the user change the option to start a sequence with 0 to 1 in-live, we
+  // delete all cell and frame 0 of the actual Xsheet
+  if (!Preferences::instance()->isSequenceCanStartWith0() &&
+      cell.getFrameId().getNumber() == 0) {
+    cell.getSimpleLevel()->eraseFrame(TFrameId(0));
```

Comment:
Good point about the X-sheet non active. I really don't think that case. :D 

I'll check out make this a reboot setting as you suggest. But, instead of reset Opentoonz, do you see accesible only closing and opening the actual scene programatically?

Agree with the confirmation popup. I din't like the deleting, too. I thought in a re-numbering instead of the deleting. I think that this functionality is made... but I don't know if it is interesting do it that way or not.

I'm going to re-think this a bit. I'm setting this again to WIP status, because I don't know when I'll get the time to put on this.